UnitsNet/UnitSystem.cs

```diff
@@ -88,5 +114,55 @@ public override int GetHashCode()
         /// Gets the SI unit system.
         /// </summary>
         public static UnitSystem SI { get; } = new UnitSystem(SIBaseUnits);
+
+        /// <summary>
+        ///     Gets the default unit information for the given quantity type associated with the current unit system.
+        ///     For example: the default unit of length in SI is the 'meter' abbreviated with the letter 'm'.
+        ///     It is possible to customize the associations by creating derived UnitSystems (immutable) using the method
+        ///     <see cref="WithDefaultUnit" />.
+        /// </summary>
+        /// <param name="quantityType">The quantity type of interest.</param>
+        /// <returns>
+        ///     The default UnitInfo for the given quantity type, if such an association exists,
+        ///     and <see langword="null" /> otherwise.
```

Comment:
.NET docs often write this as `The default [...] -or- null if no association exists.